src/components/Dashboard/Dashboard.js

```diff
@@ -10,13 +11,64 @@ const Dashboard = () => {
   if (!user) {
     navigate('/login')
   }
+  const menu = [
+    {
+      categoryName: "tatli",
+      items: [
+        {
+          itemName: "su",
+          itemPrice: "9",
+        },
+        {
+          itemName: "lo",
+          itemPrice: "1",
+        },
+      ],
+    },
+    {
+      categoryName: "icecek",
+      items: [
+        {
+          itemName: "su",
+          itemPrice: "9",
+        },
+        {
+          itemName: "sldsassjdflo",
+          itemPrice: "1",
+        },
+      ],
+    },
+    {
+      categoryName: "icecek",
+      items: [
+        {
+          itemName: "su",
+          itemPrice: "9",
+        },
+        {
+          itemName: "sldsassjdflo",
+          itemPrice: "1",
+        },
+      ],
+    },
+  ];
 
   return (
-    <Flex direction='column' gap='12px'>
+    <Flex direction="column" gap="12px" paddingBottom="20px">
       <Header />
       <Divider />
+      <Box padding="20px">
+        <Heading fontFamily="'Helvetica', sans-serif" color="#5F7161">
+          Menu
+        </Heading>
+        <Flex direction="column" flexWrap="wrap" gap="16px" alignItems="center">
+          {menu.map((category, index) => {
```

Comment:
```suggestion
          {menu.map((category, index) => 
            <CategoryTable key={index} category={category} />
          )}
```

Böyle maplerde returnlu kullanmayalım.